session/container.go

```diff
@@ -398,3 +398,20 @@ func (x Container) IssuedBy(id user.ID) bool {
 
 	return tokenOwner.Equals(id)
 }
+
+// CopyIssuer passes Container issuer's ID into f. Handler MUST NOT be nil.
+//
+// Makes sense only for signed Container instances. For unsigned instances,
+// CopyIssuer passes zero user.ID.
+//
+// See also Sign.
+func (x Container) CopyIssuer(f func(id user.ID)) {
```

Comment:
isnt naming confusing?

Reply:
LGTM, what do you recommend?

Reply:
There is a system requirement to create container within sessions: container owner MUST equal session owner. So I've decided to provide method to copy session issuer. Usecase:
```
var tok session.Container
var cnr container.Container

// init session...

tok.CopyIssuer(cnr.SetOwner)
```

Reply:
clarification to the above: i see no use cases when simple getter `Issuer() user.ID` with the same behavior (returning zero if the token is not signed) is not enough. could you provide examples with complex logic, please?

Reply:
> tok.CopyIssuer(cnr.SetOwner)

well, that looks short and nice but works only with value setters. ok, just wanted to say that `CopyIssuer` would be hard to find IMO, i would search for `Issuer()` as an SDK user

Reply:
I think `.Issuer() user.ID` is more understandable and common.

Reply:
Done